.github/workflows/ci.yml

```diff
@@ -131,6 +131,28 @@ jobs:
           fail_ci_if_error: false
           verbose: true
 
+  publish:
+    name: Publish crates on release
+    # Only do this job when publishing a release
+    if: github.event_name == 'release' && github.event.action == 'published'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install toolchain
+        uses: dtolnay/rust-toolchain@stable
```

Comment:
I forget; do we specify `rust-version`? If so, this needs to be that toolchain version.